.github/workflows/getEnvironmentVariables.js

```diff
@@ -194,9 +194,7 @@ async function checkIfOldVersion(version, token) {
 }
 
 async function getTheNewestVersion(token) {
-  const octokit = new Octokit({
-    auth: token,
-  });
+  const octokit = new Octokit();
```

Comment:
Is this change expected? `token` is no longer used

Reply:
No, I removed it ti test it. Change reverted